tests/integration/component.go

```diff
@@ -190,15 +185,13 @@ func componentTests(args ...string) {
 			Expect(desiredCompListJSON).Should(MatchJSON(actualCompListJSON))
 			cmpAllList := helper.CmdShouldPass("odo", append(args, "list", "--all")...)
 			Expect(cmpAllList).To(ContainSubstring("cmp-git"))
-			helper.CmdShouldPass("odo", append(args, "delete", "cmp-git", "-f")...)
 		})
 
 		It("should list the component when it is not pushed", func() {
 			helper.CmdShouldPass("odo", append(args, "create", "nodejs", "cmp-git", "--project", project, "--git", "https://github.com/openshift/nodejs-ex", "--min-memory", "100Mi", "--max-memory", "300Mi", "--min-cpu", "0.1", "--max-cpu", "2", "--context", context, "--app", "testing")...)
 			cmpList := helper.CmdShouldPass("odo", append(args, "list", "--context", context)...)
 			Expect(cmpList).To(ContainSubstring("cmp-git"))
 			Expect(cmpList).To(ContainSubstring("Not Pushed"))
-			helper.CmdShouldPass("odo", append(args, "delete", "-f", "--all", "--context", context)...)
```

Comment:
https://github.com/openshift/odo/pull/2312/files#r337863354, however more validation steps can be added here. For example 
```
helper.CmdShouldFail("odo", append(args, "delete", "-f")
helper.CmdShouldPass("odo", append(args, "delete", "-f", "--all", "--context", context)...)
Add a steps that should verify there should not be any odo local config context
```